api/lib/reducers/polkadotV0-reducers.js

```diff
@@ -597,17 +589,13 @@ function tallyReducer(network, tally, totalIssuance) {
   const turnout = BigNumber(tally.turnout)
 
   const totalVoted = BigNumber(tally.ayes).plus(tally.nays)
-  const total = toViewDenom(
-    network,
-    totalVoted.toString(10),
-    network.stakingDenom
-  )
-  const yes = toViewDenom(network, tally.ayes, network.stakingDenom)
-  const no = toViewDenom(network, tally.nays, network.stakingDenom)
+  const total = toViewDenom(network, totalVoted.toString(10))
+  const yes = toViewDenom(network, tally.ayes)
+  const no = toViewDenom(network, tally.nays)
   const totalVotedPercentage = turnout
     .div(BigNumber(totalIssuance))
     .toNumber()
-    .toFixed(4) // the percent conversion is done in the FE. No need to multiply by 100
+    .toFixed(4) // the percent conversion is done in the FE. We just send the decimals here
```

Comment:
don't make the FE do anything if possible

Reply:
OK. We already have this `percent` filter being used in PageProposal, but we can change this later :shrug: